src/builddef.c

```diff
@@ -101,6 +101,8 @@ int main(int argc, char **argv) {
     while ( fgets(line, MAX_LINE_LEN, bootdef_fp) ) {
         char *bootdef_key;
 
+        chomp_comments(line);
+
         if ( line[0] == '%' ) { // We hit a section, stop parsing for keyword tags
             break;
```

Comment:
@vsoch I'm not completely familiar with this code but this check seems to imply that sections like `%runscript` are not parsed by this code, so my changes shouldn't affect them.
Do you still think this could create problems with `#!/bin/bash` inside `%runscript` or similar? Keep in mind that I haven't had time to test this yet.